components/AccountImage.tsx

```diff
@@ -1,28 +1,47 @@
 /** @jsxImportSource theme-ui */
-import useAppConfig from "hooks/useConnectedAppConfig"
+import * as fcl from "@onflow/fcl"
 import {avatar} from "src/avatar"
 import {ThemeUICSSObject} from "theme-ui"
 
 interface Props {
-  address: string
+  address?: string
+  src?: string
+  seed: string
   sxStyles?: ThemeUICSSObject
+  lg?: boolean
 }
 
 const styles = {
   border: "1px solid",
```

Comment:
Should we always be using the sXStyles format?

Reply:
@10thfloor it's only required when certain props such as textTransform are used. I think it's caused by type widening when assigning styles to a const instead of inlining them.